src/test/librbd/fsx.cc

```diff
@@ -2393,12 +2393,16 @@ test(void)
 			log4(OP_SKIPPED, OP_WRITESAME, offset, size);
 			goto out;
 		}
+		break;
         case OP_COMPARE_AND_WRITE:
                 /* compare_and_write not implemented */
                 if (!ops->compare_and_write) {
                         log4(OP_SKIPPED, OP_COMPARE_AND_WRITE, offset, size);
                         goto out;
                 }
+		break;
+	default:
```

Comment:
@scienceluo 
But in L2457, as per the current implementation, the default case handles the newly added enum as an `unknown operation`.